main/demo_tasks/ota_over_mqtt_demo/ota_over_mqtt_demo.h

```diff
@@ -48,7 +52,11 @@ void vStartOTACodeSigningDemo( void );
  *
  * @return true if the message is processed by OTA.
  */
-bool vOTAProcessMessage( void * pvIncomingPublishCallbackContext,
-                         MQTTPublishInfo_t * pxPublishInfo );
+    bool vOTAProcessMessage( void * pvIncomingPublishCallbackContext,
+                             MQTTPublishInfo_t * pxPublishInfo );
+
+    #ifdef __cplusplus
+}
+    #endif
```

Comment:
Wrap these in INDENT_OFF and INDENT_ON so that the formatting looks correct.
See an example [here](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/blob/4471af64de1d481374881917bbba8736b1217530/source/include/FreeRTOS_ND.h#L37-L41) and [here](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/blob/4471af64de1d481374881917bbba8736b1217530/source/include/FreeRTOS_ND.h#L37-L41).